pkg/api/client.go

```diff
@@ -38,6 +38,15 @@ type ClientOptions struct {
 	// Default is no logging.
 	Log io.Writer
 
+	// LogIgnoreEnv disables respecting the GH_DEBUG environment variable for logging.
+	LogIgnoreEnv bool
+
+	// LogColorize enables colorized logging to Log for display in a terminal.
+	LogColorize bool
+
+	// LogVerboseHTTP enables logging HTTP headers and bodies.
```

Comment:
```suggestion
	// LogVerboseHTTP enables logging HTTP headers and bodies.
	// Default is only logging request URLs and response statuses.
```

I am actually unsure what httpretty.Logger does by default but would like to see it described here in the comments.